src/applications/disability-benefits/all-claims/tests/fixtures/data/full-781-781a-8940-test.json

```diff
@@ -191,7 +191,7 @@
     ],
     "newDisabilities": [
       {
-        "condition": "PTSD (post traumatic stress disorder)",
+        "condition": "PTSD (post-traumatic stress disorder), combat-related",
```

Comment:
I think this test data includes form 781a, which is related to PTSD, personal assault. So maybe another `newDisabilities` entry is needed for this?

Reply:
`PTSD (post traumatic stress disorder)`
has been replaced by `'PTSD (post-traumatic stress disorder), combat-related'` and
`'PTSD (post-traumatic stress disorder), non-combat-related'` in [disabilityLabelsRevised.js](https://github.com/department-of-veterans-affairs/vets-website/blob/feature-revised-526-labels-prod-wip-testing/src/applications/disability-benefits/all-claims/content/disabilityLabelsRevised.js#L402)